airflow/Dockerfile

```diff
@@ -60,32 +60,24 @@ RUN microdnf install -y yum python${PYTHON} \
 ENV HOME=/stackable
 ENV AIRFLOW_USER_HOME_DIR=/stackable
 ENV PATH=$PATH:/bin:$HOME/app/bin
-ENV AIRFLOW_UID="50000"
+ENV AIRFLOW_UID="1000"
```

Comment:
Does this require a manual migration step for users?

Reply:
The data should not be affected by this change. But the UID stored in the airflow-operator must be in sync with this one. So the airflow-operator 23.4.0 is only compatible with the stackable23.4.0 Airflow image.

Reply:
Alright, fair enough.